### PR TITLE
Remove autostart plug

### PR DESCRIPTION
**Pull request by bartoszWojciechO**

### CODE REVIEW

File: internal/constants.go [hunk 1]
```diff
@@ -178,7 +178,7 @@ func GetFilesharedPid(uid int) string {
 // GetConfigDirPath returns the directory used to store local user config and logs
 func GetConfigDirPath(homeDirectory string) (string, error) {
 	// USER_DATA is set in case of snap.
-	snapUserDataDir := os.Getenv("USER_DATA")
+	snapUserDataDir := os.Getenv("USER_DATA_COMMON")
```
Comment (Savolro) [inline internal/constants.go:181]: Since this is snap specific, `SNAP_USER_COMMON` can be used dirrectly

File: cmd/norduser/main.go [hunk 2]
```diff
@@ -47,7 +47,8 @@ func addAutostart(userHomeDir string) (string, error) {
 		"\nComment=This is an autostart for NordVPN user daemon" +
 		"\nCategories=Utility;"
 
-	path := path.Join(userHomeDir, ".config", "autostart", "nordvpn.desktop")
+	dataDir := os.Getenv("USER_DATA")
```
Comment (Savolro) [inline cmd/norduser/main.go:50]: Since this is snap specific, `SNAP_USER_DATA` can be used dirrectly

File: snap/snapcraft.yaml [hunk 3]
```diff
@@ -50,25 +50,20 @@ description: |
...
   nordvpn:
     command: bin/nordvpn
+    autostart: nordvpn.desktop
     environment:
       PREFIX_COMMON: $SNAP_COMMON
       PREFIX_DATA: $SNAP_DATA
       PREFIX_STATIC: $SNAP
-      USER_DATA: $SNAP_USER_COMMON
+      USER_DATA_COMMON: $SNAP_USER_COMMON
+      USER_DATA: $SNAP_USER_DATA
```
Comment (Savolro) [inline snap/snapcraft.yaml:62]: I think these can be removed

**Review by Savolro — CHANGES_REQUESTED** [review]

**Review by Savolro — CHANGES_REQUESTED** [review]
`snapconf.InterfaceNorduserAutostart` and its references need to be removed as well.

**Review by keliramu — APPROVED** [review]

**Review by Savolro — APPROVED** [review]

**Pull request merged**